### PR TITLE
Fix values being unquoted

### LINKED ISSUE #77: continue-config.yml parses 0 prefixed strings in env vars as numbers
## Orb version:
```
orbs:
  path-filtering: circleci/path-filtering@0.1.5
```

## What happened:

In a normal Circle CI config.yml I have a workflow like:

```
jobs:
  cypress:
    machine:
      image: ubuntu-2204:2023.02.1
    environment:
      LOCAL_SEED_ADDRESS: '0x21DF875603Ba765a15F9ae20e633D2DAe42952d9'
    steps:
      - checkout
      - node/install:
          install-yarn: true
          node-version: '16.18'
      - node/install-packages:
          pkg-manager: yarn
      - run: echo $LOCAL_SEED_ADDRESS
```

This returns the variable as expected.

However, if when I moved this same job file to a continue-config.yml and run it using the path-filtering orb, when it is invoked, I am getting:

image

Which looks like this string is being interpreted as a hex number:

image

Example Circle Job where this happened: https://app.circleci.com/pipelines/github/coordinape/coordinape/375/workflows/35b199a6-562f-4b46-a413-f3067e61977b/jobs/979

## Expected behavior:

The expected behavior is that echo'ing this env var should give me back a string of 0x21DF875603Ba765a15F9ae20e633D2DAe42952d9 not a hex number version of that string.

Like it does in the `continue.yml` job here: https://app.circleci.com/pipelines/github/coordinape/coordinape/388/workflows/5fe5261c-3e03-4847-a5b9-0c76aeb55782/jobs/1033
## Additional Information:

Here is a branch with the full config circle ci config files to reproduce this issue: https://github.com/coordinape/coordinape/tree/circle-ci-path-filtering-bug-repro with configs: https://github.com/coordinape/coordinape/tree/circle-ci-path-filtering-bug-repro/.circleci

### PR DESCRIPTION
**Pull request by Fernando-Abreu**
Fixes #77 
Cue library removes quotations from config values. This has caused issues for a customer with a hex string being transformed into a float number. 
YQ library doesn't alter quotations by default.

### CODE REVIEW

File: src/scripts/generate-config.sh [hunk 1]
```diff
@@ -1,22 +1,22 @@
 #!/usr/bin/env bash
 
 # GitHub's URL for the latest release, will redirect.
-GITHUB_BASE_URL="https://github.com/cue-lang/cue"
+GITHUB_BASE_URL="https://github.com/mikefarah/yq"
 LATEST_URL="${GITHUB_BASE_URL}/releases/latest/"
 DESTDIR="${DESTDIR:-/usr/local/bin}"
 
 function installCue() {
```
Comment (h42zhu) [inline src/scripts/generate-config.sh:8]: \~~ rename the function to installYq

**Review by h42zhu — APPROVED** [review]

File: src/scripts/generate-config.sh [hunk 2]
```diff
@@ -53,9 +53,11 @@ function generateConfig() {
 
   touch "${PARAM_GENERATED_CONFIG_PATH}"
 
+  # shellcheck disable=SC2154,SC2016
   < "${PARAM_CONFIG_LIST_PATH}" \
-  awk 'NF {printf "\"%s\" ", $0}' \
-  | xargs -0 -I {} sh -c 'cue export {} --out yaml' \
+  awk '{$1=$1};1' \
```
Comment (stig) [inline src/scripts/generate-config.sh:58]: What does this line do?

Comment (Fernando-Abreu) [reply src/scripts/generate-config.sh:58]: It removes leading and trailing whitespaces

Comment (stig) [reply src/scripts/generate-config.sh:58]: TIL! I had to look up how that works, and I understand it now. 

PS: [this](https://unix.stackexchange.com/a/568679) posts' explanation says to leave out the `;` -- I am not sure if it changes the action in any way.

File: src/scripts/generate-config.sh [hunk 3]
```diff
@@ -53,11 +53,13 @@ function generateConfig() {
...
+  | awk 'NF {printf "\"%s\" ", $0}' \
+  | xargs yq eval-all '. as $item ireduce ({}; . * $item )' \
```
[the remark this answers is not in the document]
Comment (Fernando-Abreu) [reply src/scripts/generate-config.sh:60]: Yes, the project has a test: https://app.circleci.com/pipelines/github/CircleCI-Public/path-filtering-orb/326/workflows/0e648318-5fdd-4c97-a9fa-1d959ed46f3d/jobs/1787

I've also tested with the dev orb that is generated, and it worked fine. Are you testing with valid yaml files?

[the remark this answers is not in the document]
Comment (Fernando-Abreu) [reply src/scripts/generate-config.sh:60]: My test branch in my personal repo: https://github.com/Fernando-Abreu/test-circle-ci/tree/test-path-filtering-address

[the remark this answers is not in the document]
Comment (stig) [reply src/scripts/generate-config.sh:55]: Right, so we had a huddle on this and it turns out my version of `yq` is 3.1.1, which is very much older (February 2020) than Fernando's (4.34.1).

**Review by stig — APPROVED** [review]

**Pull request merged**